app/views/products/show.html.erb

```diff
@@ -37,8 +37,10 @@
           <h4><%= @product.brand %></h4>
           <p><strong>Yarn weight:</strong> <%= @product.weight %>
           <br><strong>Composition:</strong> <%= @product.material %>
-          <br><strong>Color:</strong> <%= @product.color.capitalize %></p>
-          <p class="pe-5"><%= @product.description %><br>
+          <%# <br><strong>Color:</strong> <%= @product.color.capitalize</p> %>
+          <p class="pe-5"><%= @product.description %>
+          <br>
+          <br>
```

Comment:
In the previous version from line 41, there's one break, so I think that should be enough